nixos/modules/services/home-automation/home-assistant.nix

```diff
@@ -477,11 +477,6 @@ in {
           "xiaomi_ble"
           "yalexs_ble"
         ];
-        componentsUsingPing = [
-          # Components that require the capset syscall for the ping wrapper
-          "ping"
-          "wake_on_lan"
-        ];
```

Comment:
Please restore the list without `ping` here.

Reply:
> vm-test-run-home-assistant> hass # [   57.916967] audit: type=1326 audit(1695139385.306:3): auid=4294967295 uid=286 gid=286 ses=4294967295 subj=kernel pid=1426 comm="ping" exe="/nix/store/0

From the test log, the command `ping` crashed.

> Please restore the list without ping here.

Does this imply `ping` does not crash in the test?  I am a bit confused.

Reply:
Yes, `ping` does not seem to need capset any longer after this.

Reply:
`ping` from iputils needs `capset`[1][2] syscall and `setuid`[3] syscall.

`capset` and `setuid` are included by `@system-service`, but are excluded by `~@privileged`.  I have added them back.  As a result, ping does not crash.

[1]: https://github.com/iputils/iputils/blob/8d1acb6d95731ce1b2c7ee1c361bccfaa0918487/ping/ping_common.c#L114
[2]: https://github.com/iputils/iputils/blob/8d1acb6d95731ce1b2c7ee1c361bccfaa0918487/ping/ping_common.c#L118C1-L118C1
[3]: https://github.com/iputils/iputils/blob/8d1acb6d95731ce1b2c7ee1c361bccfaa0918487/ping/ping_common.c#L123

Reply:
Thanks for researching that.